.github/workflows/release-homebrew.yaml

```diff
@@ -85,8 +88,37 @@ jobs:
           brew bump-cask-pr --write-only -f --version=$FINCH_VERSION finch
       - name: Silently install
         run: |
+          export HOMEBREW_NO_INSTALL_FROM_API=1
           cd $(brew --repo homebrew/cask)
           brew install --cask ./Casks/finch.rb
+      - name: Build project
+        run: |
+          brew install go lz4 automake autoconf libtool
+          export PATH="/opt/homebrew/opt/libtool/libexec/gnubin:$PATH"
+          which libtool
```

Comment:
is this still needed?

Reply:
I saw that command `which libtool` in the ci [workflow](https://github.com/runfinch/finch/blob/main/.github/workflows/ci.yaml#L109). I guess the reason of it is to check whether libtool is installed successfully before building the project. @davidhsingyuchen Do you have insight on this? Please correct me if I'm wrong.

Reply:
Oh, yes we should remove that. This was due to the default libtool installed on macos ec2 instances not being the same version as the one we require, so that just checks that `libtool` points to the right one.